content/en/blog/falco-plugin-k8saudit-eks.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Monitoring your EKS clusters audit logs"
+linktitle: "Monitoring your EKS clusters audit logs"
+date: 2022-10-31
+author: Thomas Labarussias
+slug: k8saudit-eks-plugin
+---
+
+At the beginning of this year, we introduced a game changing feature, [the Plugins](https://falco.org/blog/falco-announcing-plugins/). They allow Falco to monitor and trigger alert for any kind of events. We have been pleased to create plugins for [Github](https://github.com/falcosecurity/plugins/tree/master/plugins/github), [AWS Cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail), [Okta](https://github.com/falcosecurity/plugins/tree/master/plugins/okta) and replaced the way to consume the Audit Logs generated by a K8S Control Plane by a [dedicated plugin](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit). We these plugin, Falco covers more and more aspect of you infrastructure and allows you to use a single syntax for rules.
```

Comment:
```suggestion
At the beginning of this year, Falco introduced a game changing feature, [Falco Plugins](https://falco.org/blog/falco-announcing-plugins/). They allow Falco to monitor and trigger alerts for any kind of event. We have since created plugins for [Github](https://github.com/falcosecurity/plugins/tree/master/plugins/github), [AWS CloudTrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail), [Okta](https://github.com/falcosecurity/plugins/tree/master/plugins/okta), and have replaced the way Falco consumes Audit Logs generated by a K8s API server through a [dedicated plugin](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit). With these plugins, Falco covers more and more aspects of your infrastructure and allows you to use a single syntax for rules.
```